templates/operations.ts.hbs

```diff
@@ -187,26 +187,24 @@
         headers[USER_AGENT_HEADER] = [headers[USER_AGENT_HEADER], USER_AGENT_VALUE].join(" ");
       }
 
-      const requestOptions = {
-        ...this.clientConfig.fetchOptions,
-        {{#if (isRequestWithPayload request)}}body: this.clientConfig.transformRequest(options.body, headers),{{/if}}
-        headers,
-        method: "{{loud method}}"
-      };
+      const response = await runFetchHelper(
```

Comment:
Were their any measurable change in the side of a generated lib before and after with this change. Looks like we are only saving 2 lines roughly per use of this template, but it might add up.

Reply:
`yarn run check:size`  for `main` branch:
```
 PASS  lib/index.cjs.js: 44.84KB < maxSize 45KB (gzip) 

 PASS  lib/index.esm.js: 44.73KB < maxSize 45KB (gzip) 

 PASS  commerce-sdk-isomorphic-with-deps.tgz: 337.06KB < maxSize 350KB (gzip)
```

`yarn run check:size`  for `ju/custom-endpoint` branch:
```
PASS  lib/index.cjs.js: 44.13KB < maxSize 45KB (gzip) 

PASS  lib/index.esm.js: 44.02KB < maxSize 45KB (gzip) 

PASS  commerce-sdk-isomorphic-with-deps.tgz: 337.43KB < maxSize 350KB (gzip) 
```

Seems like `lib/index.cjs.js` and `lib/index.esm.js` have decreased slightly, but `commerce-sdk-isomorphic-with-deps.tgz` has increased slightly 😮